src/types/DataTypes.sol

```diff
@@ -14,12 +14,12 @@ library Batch {
     /// @notice A struct encapsulating all parameters of {SablierV2LockupDynamic.createWithDelta} except for the asset.
     struct CreateWithDeltas {
         address sender;
-        address recipient;
-        uint128 totalAmount;
         bool cancelable;
         bool transferable;
-        LockupDynamic.SegmentWithDelta[] segments;
+        address recipient;
+        uint128 totalAmount;
         Broker broker;
+        LockupDynamic.SegmentWithDelta[] segments;
```

Comment:
Because that's the order we had in our latest release. The order was changed in [this commit](https://github.com/sablier-labs/v2-periphery/commit/52831a6b8474a0d12adb5c5929b0382c88fe2208).